src/pages/jobLogs/JobLogsPopover.tsx

```diff
@@ -0,0 +1,80 @@
+import { useState, useRef } from "react";
+import styled from "@emotion/styled";
+import Icon from "@leafygreen-ui/icon";
+import { uiColors } from "@leafygreen-ui/palette";
+import Popover from "@leafygreen-ui/popover";
+import TextInput from "@leafygreen-ui/text-input";
+import { size } from "constants/tokens";
+import { useOnClickOutside } from "hooks";
+import { useUpdateURLQueryParams } from "hooks/useUpdateURLQueryParams";
+
+const { gray, focus } = uiColors;
+
+export const JobLogsPopover: React.FC = () => {
```

Comment:
Is there any reason this component shouldn't be a generic LeafyGreen table filter popover with something like an `onConfirm` prop? I imagine we'll be using it a lot!

Reply:
great idea 🥰